src/test/java/converter/ArgumentToDeveloperConverterTest.java

```diff
@@ -0,0 +1,30 @@
+package converter;
+
+import org.example.constant.RequestArgument;
+import org.example.converter.ArgumentToDeveloperConverter;
+import org.example.model.Arguments;
+import org.example.model.Developer;
+import org.junit.Test;
+
+import static junit.framework.TestCase.assertEquals;
+
+public class ArgumentToDeveloperConverterTest {
+
+    @Test
+    public void testReturnDeveloperWithCorrectValues() {
```

Comment:
All test methods titles should start with `should`

Reply:
fixed